### PR TITLE
added erlangen2020 TrueDOP

### PR DESCRIPTION
**Pull request by pascatl**

### CODE REVIEW

File: sources/europe/de/Erlangen2020.geojson [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+    "properties": {
+        "id": "Erlangen-2020",
+        "name": "Erlangen TrueDOP (2020 5,0 cm)",
+        "type": "tms",
+        "category": "photo",
+        "start_date": "2020-03-28",
+        "end_date": "2020-03-28",
+        "url": "https://geodaten.erlangen.de/luftbilder/2020/{zoom}/{x}/{y}.png",
+        "max_zoom": 21,
+        "license_url": "https://geodaten.erlangen.de/luftbilder/2020/license/index.html",
```
Comment (andrewharvey) [inline sources/europe/de/Erlangen2020.geojson:13]: I'm unable to access this URL, is it accessible for you?

Comment (pascatl) [reply sources/europe/de/Erlangen2020.geojson:13]: sorry, we had some issues tonight, but could fix them now. The ressources should be available again

**Comment by pascatl** [comment]
is anything else missing?

**Comment by cmorty** [comment]
@rbuffat I already added this to Josm. Is there anything else to do?

**Comment by grischard** [comment]
Hi @cmorty! From the [checks](https://github.com/osmlab/editor-layer-index/runs/7733781589?check_suite_focus=true), it looks like the zoom levels should be 10-20 instead of 0-21, and the polygon should be drawn clockwise instead of counterclockwise.

**Comment by grischard** [comment]
Danke @pascatl!

**Pull request merged**